cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -266,6 +266,10 @@ public static final class AppFabric {
       "run.record.monitor.cleanup.interval.seconds";
     public static final String PROGRAM_LAUNCH_THREADS = "app.program.launch.threads";
     public static final String PROGRAM_KILL_THREADS = "app.program.kill.threads";
+    public static final String RUN_DATA_CLEANUP_TTL_AGE_DAYS = "app.run.records.table.ttl.age.days";
```

Comment:
nit: 'age' is kind of redundant with 'ttl'. There's also a bunch of tables we may want to eventually include in the cleanup, so I think it would be better to name this something like `app.run.records.ttl.days`, `app.run.records.ttl.frequency.hours`, `app.run.records.ttl.initial.delay.minutes`

Reply:
dropped "table" from all 3 properties and "age" from this property.